bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java

```diff
@@ -104,14 +113,39 @@ protected int getMinimalNumberOfSegments() {
     private final Logger logger = LoggerFactory.getLogger(NikobusPushButtonHandler.class);
     private final List<ImpactedModule> impactedModules = Collections.synchronizedList(new ArrayList<>());
     private @Nullable Future<?> requestUpdateFuture;
+    private Map<String, Integer> buttonIndexMap;
 
     public NikobusPushButtonHandler(Thing thing) {
         super(thing);
+
+        buttonIndexMap = Stream
+                .of(new Object[][] { { "A", 1 }, { "B", 3 }, { "C", 0 }, { "D", 2 }, { "1A", 5 }, { "1B", 7 },
+                        { "1C", 4 }, { "1D", 6 }, { "2A", 1 }, { "2B", 3 }, { "2C", 0 }, { "2D", 2 }, })
+                .collect(Collectors.toMap(p -> (String) p[0], p -> (Integer) p[1]));
```

Comment:
Just populate the map usual way, this way isn't easier to read and is much more prone to developer error.